src/BTVNanoCommissioning/helpers/definitions.py

```diff
@@ -6273,19 +6273,19 @@ def axes_name(var_input):
         elif "pfRelIso" in var:
             unit = " Rel. Iso"
         elif "btagDeepCvL" in var:
-            unit = "Jet DeepCSV CvL"
+            unit = "DeepCSV CvL"
         elif "btagDeepCvB" in var:
-            unit = "Jet DeepCSV CvB"
+            unit = "DeepCSV CvB"
         elif "btagDeepB" in var:
-            unit = "Jet DeepCSV Prob(b)"
+            unit = "DeepCSV Prob(b)"
         elif "btagDeepC" in var:
-            unit = "Jet DeepCSV Prob(c)"
+            unit = "DeepCSV Prob(c)"
         elif "btagDeepFlavCvL" in var:
-            unit = "Jet DeepJet CvL"
+            unit = "DeepJet CvL"
         elif "btagDeepFlavCvB" in var:
-            unit = "Jet DeepJet CvB"
+            unit = "DeepJet CvB"
         elif "btagDeepFlavB" in var:
-            unit = "Jet DeepJet Prob(b)"
+            unit = "DeepJet Prob(b)"
```

Comment:
Note: for your solution, unit in line 6288 needs to be replaced as well. 

I guess this code is to avoid having labels like `1st Jet Deep1stJet prob(b)` due to the replacement [here](https://github.com/cms-btv-pog/BTVNanoCommissioning/blob/master/src/BTVNanoCommissioning/helpers/definitions.py#L6294)? I was working on that as well, my workaround would be the following:
- leave the code above unchanged
- in l6294ff: do an if else condition as follows:
```
if var.endswith("0") or "jet0" in var:
            if "btagDeepFlav" not in var:
                label = label.replace("Jet", "$1^{st}$-Jet")
            else:
                label = label.replace("Jet", "$1^{st}$-Jet",1)
        elif var.endswith("1") or "jet1" in var:
            if "btagDeepFlav" not in var:
                label = label.replace("Jet", "$2^{nd}$-Jet")
            else:
                label = label.replace("Jet", "$2^{nd}$-Jet",1)
        elif var.endswith("2") or "jet2" in var:
            if "btagDeepFlav" not in var:
                label = label.replace("Jet", "$3^{rd}$-Jet")
            else:
                label = label.replace("Jet", "$3^{rd}$-Jet",1)
        elif var.endswith("3") or "jet3" in var:
            if "btagDeepFlav" not in var:
                label = label.replace("Jet", "$4^{th}$-Jet")
            else:
                label = label.replace("Jet", "$4^{th}$-Jet",1)
```
i.e. making sure that only the first Jet instance is getting replaced. As I have this already in my version implemented, I suggest reverting your changes in l. 6726-6288 and let this handle by my PR if that is okay for you?

Reply:
Yes, indeed I want to avoid the weird format `1st Jet Deep1stJet prob(b)`. I would replace with your second solution, thanks for sharing this